src/ralph/api/routers/statements.py

```diff
@@ -382,10 +384,9 @@ async def get(
 # pylint: disable=unused-argument
 async def put(
     current_user: Annotated[AuthenticatedUser, Depends(get_authenticated_user)],
-    # pylint: disable=invalid-name
-    statementId: UUID,
     statement: LaxStatement,
     background_tasks: BackgroundTasks,
+    statement_id: UUID = Query(..., alias="statementId"),
```

Comment:
Is the ellipsis mandatory ?

Reply:
Not since this [release](https://github.com/tiangolo/fastapi/releases/tag/0.78.0), I fix it.